### PR TITLE
Added aws cli and documentation

### PR DESCRIPTION
**Pull request by guyarb**

### CODE REVIEW

File: docker/awscli.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Seekret
+=======
+Docker deployment  
+Sniffer docker will passively sniff target container traffic and move the pcaps to the remote bucket.
+Supported buckets: S3.
+
+The machine must be configured to have an IAM Role access or provide ACCESS_KEY and SECRET_KEY via the configuration to the docker
```
Comment (val06) [inline docker/awscli.md:7]: please mark this as bold or some other markdown magic

File: docker/minio.env [hunk 2]
```diff
@@ -2,9 +2,8 @@ ACCESS_KEY=
 SECRET_ACCESS_KEY=
 STORAGE_PROVIDER=
 BUCKET_NAME=
-FOLDER_NAME=
```
Comment (val06) [inline docker/minio.env:5]: is this a breaking change?

File: docker/minio.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Sniffer docker will passively sniff target container traffic and move the pcaps to the remote bucket.
+Supported buckets: S3, GCS and Azure blob storage.
+
+The machine must be configured to have an IAM Role access or provide ACCESS_KEY and SECRET_KEY via the configuration to the docker  
+
+## Installation
+
+Edit the minio.env with the following values:
+
+_Bucket access parameters: (**or** use an IAM role **instead** - with AWS S3 buckets only)_
```
Comment (val06) [inline docker/minio.md:13]: remove IAM role mention

**Review by val06 — CHANGES_REQUESTED** [review]
cred

File: docker/minio.env (hunk 2 above)
Comment (guyarb) [reply docker/minio.env:5]: shouldn't be, they can still set folder name (although you prefer they will set workspace instead)

File: docker/minio.md (hunk 3 above)
Comment (guyarb) [reply docker/minio.md:13]: removed

File: docker/awscli.md (hunk 1 above)
Comment (guyarb) [reply docker/awscli.md:7]: done

File: docker/minio.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+Seekret
+=======
+Docker deployment  
+Sniffer docker will passively sniff target container traffic and move the pcaps to the remote bucket.
+Supported buckets: S3, GCS and Azure blob storage.
+
+The machine must be configured to have an IAM Role access or provide ACCESS_KEY and SECRET_KEY via the configuration to the docker
```
Comment (val06) [inline docker/minio.md:7]: remove the IAM role option

**Review by val06 — CHANGES_REQUESTED** [review]
re-cred

File: docker/minio.md (hunk 4 above)
Comment (guyarb) [reply docker/minio.md]: great catch, fixed

**Review by val06 — APPROVED** [review]

**Pull request merged**